### DISCUSSION TITLE
Installing RE on a fresh Windows 10

### DISCUSSION THREAD

**Discussion by IceflowRE** [redeclipse/base#1124] [Help and Support] [answered]
`msvcr100.dll` for curl is not installed yet.
We should ship it or show a better info message. Because the "normal" player dont know what to do with this and clicks ok. RE starts anyway so he dont know that the udpater was skipped.

**Comment by Calinou** [comment]
I'm not sure if the MSVC redistributable license allows bundling the `.dll` file directly, but you can bundle the `vcredist` installer in your own installer and run it automatically.

**Comment by lsalzman** [comment]
We go to great lengths to NOT require the MSVC redistributable installer. So rather, we need to find some way to either build curl such that it does NOT need said MSVCR dll, or use something else, like wget.

There's no simple way to "just" bundle the redistributable, because it requires running an installer and a bunch of winsxs hell. The fact that RE itself doesn't need it but curl would need it is just something of a horrible joke.

**Comment by qreeves** [comment, marked as answer]
There are MingW linked binaries out there, and an issue has already been made for this (red-eclipse/base#451). I just forgot to actually do this, so I'm assigning it to myself now.
